website/newsletters/templates/newsletters/email.html

```diff
@@ -212,6 +200,36 @@ <h2 style="{{ item_h2_style }}">{{ item.title.upper }}</h2>
         </table>
     </div>
     {% endif %}
+    {% if local_partners.all %}
```

Comment:
```suggestion
    {% if local_partners.exists %}
```

Dit is een hele zware query die niet nodig is :)